db/seeds.rb

```diff
@@ -5,9 +5,12 @@
 # DO NOT SEED OUR ACTUAL USERS THIS WAY
 # as they are not public domain
 test_users = [
-  { email: 'joel.sugarman@justice.gov.uk', auth_provider: 'azure_ad' },
+  { email: 'joel.sugarman@justice.gov.uk', auth_provider: 'azure_ad' }
 ]
 
 test_users.each do |attributes|
   User.create_or_find_by!(attributes)
+  User.create_or_find_by!(email: 'mock.azure@justice.gov.uk',
```

Comment:
This should not be within the loop as it will error if there are more that one test users

Reply:
also, this will seed it on production and staging which i don't think we should do. Ideally we would only have this user on local and any UAT environment.

Reply:
thanks 👍 , I have removed it from the loop. It shouldn't seed it on staging or production I believe unless mock_azure == true

```
User.create_or_find_by!(email: 'mock.azure@justice.gov.uk',
                        first_name: 'mock',
                        last_name: 'azure') if Rails.configuration.x.mock_azure == "true"
```

I have left the line in there for now. to seed mock.azure@justice.gov.uk, to help with testing but happy to remove completely if you think that would be better.